lib/asciidoc-preview-view.coffee

```diff
@@ -111,9 +111,21 @@ class AsciiDocPreviewView extends ScrollView
       saveOnly = atom.config.get('asciidoc-preview.renderOnSaveOnly')
       changeHandler() if not saveOnly
 
+    scrollPreview = (event, callback) ->
+      blockId = renderer.getBlockId event.newBufferPosition.row
+      if blockId is undefined
```

Comment:
The method `getBlockId` does not return an explicit value when `exports.blocksPositions` is undefined. Is this a good practice in Coffeescript ?

The method will also return undefined if there's no block for the cursor line.

So `typeof blockId is 'string'` may be redundant since `renderer.getBlockId` will always return a string (a block id) or undefined ?